src/main/java/org/truffleruby/language/globals/AliasGlobalVarNode.java

```diff
@@ -35,7 +35,8 @@ public Object execute(VirtualFrame frame) {
     @TruffleBoundary
     private void checkExisting() {
         if (getContext().getCoreLibrary().getGlobalVariables().contains(newName)) {
-            throw new RaiseException(coreExceptions().internalError(String.format("%s is already a global", newName), this));
+            // TODO CS 4-Apr-18 This exception is non-standard
+            throw new RaiseException(coreExceptions().internalError(String.format("%s is already a global", newName), this), true);
```

Comment:
This seems OK as a RuntimeError.

Reply:
Or maybe a `NameError`, but that's usually when it's missing, not conflicting.

Reply:
I'm using `NotImplementedError`.